### ISSUE TITLE
Pkgconfig from version 1.0 file does not match installed library name and include directory path.

### ISSUE THREAD

**Issue by darkshram** [manisandro/gImageReader#556]
As default include directory is set as `/usr/include/QtSpell-qt5`  and the library name as `libqtspell-qt5.so`, but `QtSpell-qt5.pc` installs with these contents:

```
prefix=/usr
libdir=${prefix}/lib
includedir=${prefix}/include/QtSpell-5

Name: qtspell
Description: Spell checking for Qt text widgets.
Version: 1.0
Libs: -L${libdir} -lqtspell-5
Cflags: -I${includedir}
```
Neither the `includedir` or the `Libs` values match with the installed files. So, gimagereader fails to build with qtspell 1.0, unless `QtSpell-qt5.pc` is modified:

```
prefix=/usr
libdir=${prefix}/lib
includedir=${prefix}/include/QtSpell-qt5

Name: qtspell
Description: Spell checking for Qt text widgets.
Version: 1.0
Libs: -L${libdir} -lqtspell-qt5
Cflags: -I${includedir}
```
Same thing happens if qtspell is build with Qt6 support. `QtSpell-qt6.pc` content does not match with the real include directory and the library name to link.

**Comment by manisandro** [comment]
Yes already fixed in 1.0.1, apologies for the mistake.

**Comment by darkshram** [comment]
Thank you! I'm sorry for posting it here. Should have been in qtspell issues.
